api/managementpb/checks.proto

```diff
@@ -33,10 +33,29 @@ message SecurityCheckResult {
   map<string, string> labels = 4;
 }
 
+// SecurityCheck contains check name and status.
+message SecurityCheck {
+  string name = 1;
+  bool disabled = 2;
+}
+
 message GetSecurityCheckResultsResponse {
   repeated SecurityCheckResult results = 1;
 }
 
+message ListSecurityChecksRequest {}
+
+message ListSecurityChecksResponse {
+  repeated SecurityCheck checks = 1;
+}
+
+message ToggleSecurityChecksRequest {
```

Comment:
I'm not sure that this request will let user disable or enable particular check

Reply:
Hm, why?  You pass names of checks that you want to enable/disable, what's the problem here?

Reply:
oh, ok. looks weird, but ok.

Reply:
If you have an idea of how to make that API better - please share, otherwise, we will use that one FOREVER! 😈

Reply:
list of messages with three fields(name, disable, enable)?

Reply:
Done, check it please